rasa/cli/data.py

```diff
@@ -230,6 +233,8 @@ def _convert_nlu_data(args: argparse.Namespace) -> None:
         NLUMarkdownToYamlConverter,
     )
 
+    rasa.utils.common.run_in_loop(telemetry.track_data_convert(args.format, "nlu"))
```

Comment:
We track the "convert" events **before** we start the conversion, while the data others in here are track **after** the operations are done. Would be great to have this consistent.

Reply:
yes will change that :+1: